src/nora/components/NoraButton/NoraButton.module.scss

```diff
@@ -0,0 +1,17 @@
+.Button {
+  font-weight: 500;
+  font-size: 13px;
```

Comment:
We have a compact font that's been used for Nora so far and it's 15 px. If we want to add a 13px (I presume you verified some figma designs or something), then we should figure out a way to do that in a sharable way across other components that'll need it so we have a sort of system if you will. Maybe it's  18px | 15px | 13px or whatever. But I'd prefer we centralize it somehow.

Reply:
Hm, good to know. I just verified that the mocks have 13px for buttons (it seems like when things are bolded the font sometimes is smaller), so perhaps we can bring this up with Tom.

Would you prefer for me to change it to 15 px for now?

Reply:
No, I think 13px is probably correct. I'd just like to have it defined so we can do something like `font-size: var(--x-compressed-font-size) * 1px` or something  similar 🤔It's a bit hard...we already conveniently had the compact one which I repurposed for Nora's main font size, but, we'll need to add one for the 13px variant.

Reply:
I'll add a 13px font attribute to the `Attributes.scss` file that we have for Nora, and perhaps as a follow-up action, we can take a holistic look at this with Tom?